source/ranges.tex

```diff
@@ -12881,3 +12891,615 @@
+\pnum
+\effects
+Equivalent to:
+\begin{codeblock}
+auto r = i;
+r += n;
+return r;
+\end{codeblock}
+\end{itemdescr}
+
+\indexlibrarymember{operator-}{stride_view::iterator}%
+\begin{itemdecl}
+friend constexpr @\exposid{iterator}@ operator-(const @\exposid{iterator}@& i, difference_type n)
+  requires @\libconcept{random_access_range}@<@\exposid{Base}@>;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to:
+\begin{codeblock}
+auto r = i;
+r -= n;
+return r;
+\end{codeblock}
+\end{itemdescr}
+
+\indexlibrarymember{operator-}{stride_view::iterator}%
+\begin{itemdecl}
+friend constexpr difference_type operator-(const @\exposid{iterator}@& x, const @\exposid{iterator}@& y)
+  requires @\libconcept{sized_sentinel_for}@<iterator_t<@\exposid{Base}@>, iterator_t<@\exposid{Base}@>>;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\returns
+Let \tcode{N} be \tcode{x.\exposid{current_} - y.\exposid{current_}}.
+\begin{itemize}
+\item
+If \exposid{Base} models \libconcept{forward_range},
+\tcode{(N + x.\exposid{missing_} - y.\exposid{missing_}) / x.\exposid{stride_}}.
+\item
+Otherwise, if \tcode{N} is negative, \tcode{-\exposid{div-ceil}(-N, x.\exposid{stride_})}.
+\item
+Otherwise, \tcode{\exposid{div-ceil}(N, x.\exposid{stride_})}.
+\end{itemize}
+\end{itemdescr}
+
+\indexlibrarymember{operator-}{stride_view::iterator}%
+\begin{itemdecl}
+friend constexpr difference_type operator-(default_sentinel_t y, const @\exposid{iterator}@& x)
+  requires @\libconcept{sized_sentinel_for}@<sentinel_t<@\exposid{Base}@>, iterator_t<@\exposid{Base}@>>;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\returns
+\tcode{\exposid{div-ceil}(x.\exposid{end_} - x.\exposid{current_}, x.\exposid{stride_})}.
+\end{itemdescr}
+
+\indexlibrarymember{operator-}{stride_view::iterator}%
+\begin{itemdecl}
+friend constexpr difference_type operator-(const @\exposid{iterator}@& x, default_sentinel_t y)
+  requires @\libconcept{sized_sentinel_for}@<sentinel_t<@\exposid{Base}@>, iterator_t<@\exposid{Base}@>>;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to: \tcode{return -(y - x);}
+\end{itemdescr}
+
+\indexlibrarymember{iter_move}{stride_view::iterator}%
+\begin{itemdecl}
+friend constexpr range_rvalue_reference_t<@\exposid{Base}@> iter_move(const @\exposid{iterator}@& i)
+  noexcept(noexcept(ranges::iter_move(i.@\exposid{current_}@)));
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to: \tcode{return ranges::iter_move(i.\exposid{current_});}
+\end{itemdescr}
+
+\indexlibrarymember{iter_swap}{stride_view::iterator}%
+\begin{itemdecl}
+friend constexpr void iter_swap(const @\exposid{iterator}@& x, const @\exposid{iterator}@& y)
+  noexcept(noexcept(ranges::iter_swap(x.@\exposid{current_}@, y.@\exposid{current_}@)))
+  requires @\libconcept{indirectly_swappable}@<iterator_t<@\exposid{Base}@>>;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to:
+\tcode{ranges::iter_swap(x.\exposid{current_}, y.\exposid{current_});}
```

Comment:
```suggestion
\tcode{ranges::iter_swap(x.\exposid{current_}, y.\exposid{current_})}
```

Reply:
That's a "statement"-level "equivalent to", and the semicolon is part of the statement.

Reply:
They should be equivalent. And I forgot to add the period to my suggestion.

Reply:
I think we allow expressions of void type to be used in sentence style, "Equivalent to `foo()`.", but I don't think we _have_ to do that, especially if there are a lot of similar nearby cases of non-void expressions where we want the colon form. I think it's fine to leave as is unless someone can make a strong case for why we should use sentence style.